apps/avatax/src/modules/avatax/calculate-taxes/avatax-calculate-taxes-adapter.ts

```diff
@@ -45,10 +103,10 @@ export class AvataxCalculateTaxesAdapter
 
       return transformedResponse;
     } catch (e) {
-      const error = normalizeAvaTaxError(e);
+      const error = this.parseAvataxError(e);
 
       /**
-       * TODO: Add handling of specific cases
+       * TODO: Refactor errors so we are able to print error only for unhandled cases, otherwise use warnings etc
        */
       this.logger.error("Error calculating taxes", { error });
```

Comment:
question: do we need this logger here if down the line we also have logs?

e.g.:
```
logger.warn(
              "InvalidAppAddressError: App returns status 400 due to broken address configuration",
              { error },
            );
```

Reply:
Not sure if I get where. I prefer to observe it in datadog and have more errors at the beginning, then reduce them